code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -1132,3 +1132,30 @@
 		M.electrocute_act(rand(5,20), "Teslium in their body", 1, 1) //Override because it's caused from INSIDE of you
 		playsound(M, "sparks", 50, 1)
 	..()
+
+/datum/reagent/peaceborg/confuse
+	name = "Dizzying Solution"
+	id = "dizzysolution"
+	description = "Makes the target off balance and dizzy"
+	metabolization_rate = 1.5 * REAGENTS_METABOLISM
+
+/datum/reagent/peaceborg/confuse/on_mob_life(mob/living/M)
+	M.AdjustConfused(3, bound_lower = 0, bound_upper = 5)
+	M.AdjustDizzy(3, bound_lower = 0, bound_upper = 5)
+	if(prob(20))
+		to_chat(M, "<span class='warning'>You feel confused and disorientated.</span>")
+	..()
+
+/datum/reagent/peaceborg/tire
+	name = "Tiring Solution"
+	id = "tiresolution"
+	description = "An extremely weak stamina-toxin that tires out the target. Completely harmless."
+	metabolization_rate = 1.5 * REAGENTS_METABOLISM
+
+/datum/reagent/peaceborg/tire/on_mob_life(mob/living/M)
+	var/healthcomp = (M.maxHealth - M.health)
+	if(M.staminaloss < (45 - healthcomp))	//At 50 health you would have 200 - 150 health meaning 50 compensation. 60 - 50 = 10, so would only do 10-19 stamina.)
+		M.adjustStaminaLoss(10)
+	if(prob(30))
+		to_chat(M, "<span class='warning'>You should sit down and take a rest...</span>")
```

Comment:
works better as
`You feel you should sit down and take a rest...`

As you're trying to get across how the guy is feeling, not giving him an order...